packages/plugin-e2e/playwright.config.ts

```diff
@@ -38,14 +38,23 @@ export default defineConfig<PluginOptions>({
       name: 'authenticate',
       testMatch: [/.*auth\.setup\.ts/],
     },
-    // 2. Run all tests in parallel with Chrome.
+    // 2. Create a datasource that can be used across tests (should use provsiioning DS instead, but currently not working in CI)
+    {
```

Comment:
This is not ideal. I'd much rather have the data source being provisioned instead. For internal plugins, this will be straightforward. They can just clone the plugin-provisioning repo and mount it as a volume when starting up the grafana container. It's trickier for community plugins...me and Brain talked about adding a `provisioning` folder to the root of the plugin, and then mount this folder in the Grafana docker container. Possibly we could create a GH action that also expands secrets defined in the YAML. Anyway, this will be something to tackle in a future PR.